book/src/usage.md

```diff
@@ -145,25 +165,25 @@ though, we climb the syntax tree and then take the previous selection. So
 | `t`                    | Test                     |
 | `g`                    | Change                   |
 
-> NOTE: `f`, `c`, etc need a tree-sitter grammar active for the current
+> 💡 `f`, `c`, etc need a tree-sitter grammar active for the current
 document and a special tree-sitter query file to work properly. [Only
 some grammars][lang-support] currently have the query file implemented.
 Contributions are welcome!
 
-## Tree-sitter Textobject Based Navigation
+## Navigating Using Tree-sitter Textobject
 
-Navigating between functions, classes, parameters, etc is made
-possible by leveraging tree-sitter and textobjects queries. For
+Navigating between functions, classes, parameters, and other elements is
+possible using tree-sitter and Textobject queries. For
```

Comment:
In sentences here I think it's ok to call these "textobject" rather than "Textobject"

Reply:
I have done that for all instances in sentences, and left when in headings / first word in sentences